website/orders/migrations/0002_auto_20200505_1243.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.0.5 on 2020-05-05 10:43
+
+import datetime
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("orders", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="shift",
+            name="end_date",
+            field=models.DateTimeField(default=datetime.datetime(2020, 5, 5, 13, 15)),
```

Comment:
Why is this the default value? It appears to be a Django bug maybe with the auto_now setting for these fields?

Reply:
Hmm weird, will take a look